pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -53,19 +54,19 @@ stdenv.mkDerivation (finalAttrs: {
     + lib.optionalString hostCpuOnly "-host-cpu-only"
     + lib.optionalString nixosTestRunner "-for-vm-tests"
     + lib.optionalString toolsOnly "-utils";
-  version = "8.1.3";
+  version = "8.2.0-rc3";
 
   src = fetchurl {
```

Comment:
would there be a tangible advantage when using the GitHub mirror at https://github.com/qemu/qemu, and switching to `fetchFromGitHub` ?

Reply:
No.  First, building releases is what the release tarballs are for, and second, why would we use GitHub and not the actual upstream repository on GitLab?

Reply:
That makes sense, thank you for the additional info!